docs/extras/blocking_nintendo.md

```diff
@@ -3,24 +3,24 @@
 This page will help you set up a method to block all communication with Nintendo. This will stop any updates and reporting to Nintendo, but it will prevent use of the eShop and online games.
 
 ### Instructions (AMS DNS redirection)
-You can configure Atmosphere to automatically redirect any requests directed to Nintendo to nothing instead. Documentation about this feature can be [found here](https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/dns_mitm.md). The section below will help you set up DNS redirection on your emummc. Note that this will only apply when you are using cfw.
+You can configure Atmosphere to automatically redirect any requests directed to Nintendo to nothing instead. Documentation about this feature can be [found here](https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/dns_mitm.md). The section below will help you set up DNS redirection on your emuMMC. Note that this will only apply when you are using CFW.
 
-#### What you need: 
+#### What you need:
 
 - <a href="../../files/emummc.txt" download>emummc.txt</a>
 - Atmosphere 0.18.0 or higher
 
 #### Setup:
 
-1. Insert your SD into your pc
+1. Insert your SD into your PC
```

Comment:
If we're changing SD to microSD Card in other pages it could probably be done here as well.